easybuild/easyconfigs/i/ImageMagick/ImageMagick-6.9.3-2-goolf-1.4.10.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'ImageMagick'
+version = '6.9.3-2'
```

Comment:
this version seems to have disappeared, now there's only `6.9.3-3` available...

the version from #1522 (`6.9.0-9`) is also gone (now only `6.9.0-10` is there)

not sure how to handle this... is there an alternative download location?